HyperV/scripts/gerrit-git-prep.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash -e

```

Comment:
Please update it based on the latest upstream version: https://github.com/openstack-infra/project-config/blob/master/jenkins/scripts/gerrit-git-prep.sh
We've added mainly some logging on top of that.